### PR TITLE
Implementing Frontend Interface

### PR DESCRIPTION
**Pull request by eduardorrramos**
**Description**

- Cleaning code, polishing CSS.
- Created database to store images for each border crossing: fetch call fails to retrieve relevant photo for every single location.
- Implemented Geolocation into my emergency alert.

**Upcoming**: 

- Continue polishing/cleaning my code.

**Milestones**

- User Experience, as I come towards the end of my project, I am largely wrapping everything together and testing.

### CODE REVIEW

File: capstone-frontend/src/App.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #root {
-  max-width: 1280px;
+  /* max-width: 1280px; */
```
Comment (kenamehta) [inline capstone-frontend/src/App.css:2]: remove if not needed

File: capstone-frontend/src/App.css [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   margin: 0 auto;
-  padding: 2rem;
+  /* padding: 2rem; */
```
Comment (kenamehta) [inline capstone-frontend/src/App.css:4]: remove if not needed

File: capstone-frontend/src/components/modal.jsx [hunk 3]
```diff
@@ -42,9 +62,14 @@ function ModalPopulate() {
       isOpen={modalIsOpen}
       onRequestClose={closeModal}
       style={customStyles}
-      contentLabel="Example Modal"
+      contentLabel="Emergency Modal"
     >
-      This alert was sent by {userName} at 
+      This alert was sent by {userName} at {userLongitude}, {userLatitude}!
+       {/* <APIProvider apiKey={apiKey}>
```
Comment (kenamehta) [inline capstone-frontend/src/components/modal.jsx:68]: remove if not needed

**Comment by kenamehta** [comment]
@eduardorrramos Since it is a UI change, could you provide a link to the application's UI and its functioning. This will help the reviewers view the changes made in the UI

**Review by kenamehta — COMMENTED** [review]
Left a comment on adding link for working application's screenrecording. This will greatly help review this pullrequest

File: capstone-frontend/src/App.css (hunk 2 above)
Comment (eduardorrramos) [reply capstone-frontend/src/App.css:4]: Got it!

**Pull request merged**